config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
I was mainly referring to a single source of truth inside this page to avoid errors. Performance was not the issue :)